.github/workflows/generate-blog.yml

```
@@ -0,0 +1,38 @@
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Add this to ensure git history is available
+      
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          
+      - name: Install dependencies
+        run: npm ci
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)